src/test/java/se/liu/ida/hefquin/engine/wrappers/lpgwrapper/impl/LPG2RDFConfigurationReaderTest.java

```diff
@@ -0,0 +1,144 @@
+package se.liu.ida.hefquin.engine.wrappers.lpgwrapper.impl;
+
+
+import org.apache.jena.graph.Node;
+import org.apache.jena.rdf.model.Model;
+import org.apache.jena.rdf.model.ModelFactory;
+import org.apache.jena.riot.Lang;
+import org.apache.jena.riot.RDFParser;
+import org.apache.jena.riot.RDFParserBuilder;
+import org.junit.Test;
+import se.liu.ida.hefquin.engine.wrappers.lpgwrapper.data.impl.LPGNode;
+
+import static org.junit.Assert.*;
+
+public class LPG2RDFConfigurationReaderTest {
+
+    @Test
+    public void LPG2RDFConfigWithIRIBasedNodeMapping() {
+        final String turtle =
+                "PREFIX lr:     <http://www.example.org/se/liu/ida/hefquin/lpg2rdf#>\n"
+              + "PREFIX ex:     <http://example.org/>\n"
+              + "\n"
+              + "ex:LPGtoRDFConfig\n"
+              + "   a  lr:LPGtoRDFConfiguration ;\n"
+              + "   lr:labelPredicate  <http://www.w3.org/2000/01/rdf-schema#label> ;\n"
+              + "   lr:nodeMapping  ex:IRINodeMapping ."
+              + "\n"
+              + "ex:IRINodeMapping\n"
+              + "   a  lr:IRIBasedNodeMapping ;\n"
+              + "   lr:prefixOfIRIs  <https://example.org/node/> .";
```

Comment:
Same issue here, the object of this triple must be a literal:
```suggestion
              + "   lr:prefixOfIRIs  "https://example.org/node/"^^xsd:anyURI .";
```